.github/workflows/on-pr.yml

```diff
@@ -0,0 +1,41 @@
+name: "Validate and plan"
+
+on:
+  pull_request:
+
+jobs:
+  test:
+    runs-on: ubuntu-18.04
+    strategy:
+      matrix:
+        java-version: [ 8 ]
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-java@v2
+        with:
+          distribution: 'adopt'
+          java-version: ${{ matrix.java-version }}
+          cache: 'maven'
+      - name: Configure git user
+        run: |
+          git config user.email "actions@github.com"
+          git config user.name "GitHub Actions"
+      - name: Build and test
+        run: |
+          mvn clean install -Dmaven.javadoc.skip --quiet
```

Comment:
Why skip javadoc? I don't think there's anything configured to build it on install.